code/modules/random_map/drop/drop_types.dm

```diff
@@ -83,8 +83,8 @@ var/global/list/datum/supply_drop_loot/supply_drop
 		/obj/item/clothing/accessory/storage/black_drop_pouches,
 		/obj/item/weapon/storage/backpack/dufflebag/sec,
 		/obj/item/weapon/gun/projectile/automatic/carbine,
-		/obj/item/ammo_magazine/a556m/ap,
-		/obj/item/ammo_magazine/a556m,
+		/obj/item/ammo_magazine/c762/ap,
+		/obj/item/ammo_magazine/c762,
```

Comment:
If you take my earlier suggestion to leave the STS and Carbine as they are, and just switch which ones are available, then remember to put these back to the way they were.